packages/moxb/src/form/FormImpl.ts

```diff
@@ -57,6 +57,13 @@ export class FormImpl extends BindImpl<FormOptions> implements Form {
         return !!(this.impl.values.find(v => v.errors!.length > 0) || this.errors!.length > 0);
     }
 
+    @computed
+    get hasChanges() {
+        return !this.impl.values.every(v => {
+            return v.isInitialValue === undefined ? true : v.isInitialValue;
```

Comment:
...thinking about `isInitialValue` makes me wonder if that is a good name. Maybe `Value` should have a `isChanged` or `hasChanges` attribute. The problem with `isInitialValue` is that `undefined` is not the same as `false` and that makes things (like this expression) more complicated.

I added an Issue for that #39

Reply:
.. I thougth about that too, because that the `initalValue` could be `undefined` have given me a hard time to creating the `hasChanges` method. Maybe a different approach could simplify and clear out this difficulties.